### PR TITLE
Initial file to main

### PR DESCRIPTION
**Pull request by lars-cornelissen**
Created initial lua files for turtles on server

### CODE REVIEW

File: computer_code/0/data/last_farmed [hunk 1]
```diff
@@ -0,0 +1 @@
+20.0
```
Comment (lars-cornelissen) [inline computer_code/0/data/last_farmed:1]: Files should not be pushed

File: computer_code/0/data/potatoes_farmed [hunk 2]
```diff
@@ -0,0 +1 @@
+400.0
```
Comment (lars-cornelissen) [inline computer_code/0/data/potatoes_farmed:1]: Files should not be pushed

File: computer_code/0/farmPotaters.lua [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    end
+    turtle.turnRight()
+    while os.day() - global_vals.getValue("last_farmed") < wait_time do
+        term.clear()
+        print("Total farmed by sparkle: " .. potato_counter.get())
+        print("Days since last farm: " .. os.day() - global_vals.getValue("last_farmed") .. "/" .. wait_time)
+        os.sleep(5)
+    end
+end
+print("OUT OF FUEL. FEED ME BITCH")
```
Comment (lars-cornelissen) [inline computer_code/0/farmPotaters.lua:43]: Watch your profanity

File: computer_code/0/farmPotaters.lua [hunk 4]
```diff
@@ -0,0 +1,43 @@
+size_x = 9
+size_y = 9
+
+os.loadAPI("potato_counter")
+os.loadAPI("global_vals")
+wait_time = 5
+
+print("starting!")
+while turtle.getFuelLevel() > 150 do
+    global_vals.store("last_farmed", os.day())
```
Comment (lars-cornelissen) [inline computer_code/0/farmPotaters.lua:10]: maybe change variable name

File: computer_code/0/data/last_farmed (hunk 1 above)
Comment (LisaPouels) [inline computer_code/0/data/last_farmed:1]: test

**Pull request merged**